setup.py

```diff
@@ -382,7 +382,6 @@ def get_file_list (self):
         'linkcheck.configuration',
         'linkcheck.director',
         'linkcheck.htmlutil',
-        'linkcheck.HtmlParser',
```

Comment:
(At some point we ought to switch to `setuptools.find_packages()` instead of manually enumerating all the subpackages that exist.)

Reply:
Noted.